corehq/form_processor/backends/sql/dbaccessors.py

```diff
@@ -506,8 +506,9 @@ def soft_delete_forms(domain, form_ids, deletion_date=None, deletion_id=None):
     @transaction.atomic
     def _archive_unarchive_form(form, user_id, archive):
         from casexml.apps.case.xform import get_case_ids_from_form
+        from corehq.form_processor.parsers.ledgers.form import get_case_ids_from_stock_transactions
         form_id = form.form_id
-        case_ids = list(get_case_ids_from_form(form))
+        case_ids = list(get_case_ids_from_form(form) | get_case_ids_from_stock_transactions(form))
```

Comment:
This was causing a test to fail: https://github.com/dimagi/commcare-hq/blob/b98ec3f6d8063ae65e06c51f95e4fbefd23e24c8/corehq/apps/commtrack/tests/test_xml.py#L529

But it was also failing for me on master so not sure how it every passed